.github/workflows/check-lambda-js.yml

```diff
@@ -36,28 +37,30 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
-          node-version: '20.15.1'
+          node-version: '22.1.0'
 
       # Use any cached yarn dependencies (saves build time)
       - uses: actions/cache@v2
         with:
           path: '**/node_modules'
           key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
 
-      # =======================================================================
-      # =                      Cloudfront CSP Lambda                          =
```

Comment:
## UI Bug

<img width="365" alt="image" src="https://github.com/user-attachments/assets/7d98d0d7-f7bb-417f-8efa-b38f1d278a3b" />

Steps to reproduce-

1. Log in as Board ED
2. Go to `/aslp/Licensing`
3. Click "Search" to show all providers.
4. Navigate to another page, e.g., `/Account` by clicking on the "Account" menu item.
5. Navigate back to `/aslp/Licensing` by clicking on the "Licensing" menu item.